mailer/migrations/0007_create_emailtempates_id.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-02-13 11:21
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from django.conf import settings
+
+
+MAILER_SENDER_EMAILS = getattr(settings, 'MAILER_SENDER_EMAILS', ())
+MAILER_SENDER_EMAILS_CHOICES = list(zip(MAILER_SENDER_EMAILS, MAILER_SENDER_EMAILS)) if MAILER_SENDER_EMAILS else None
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mailer', '0006_auto_20170213_1221'),
+    ]
+
+    def populate_template_ids(apps, schema_editor):
+        templates = apps.get_model('email_sender', 'emailtemplate').objects.all()
+        for i, template in enumerate(templates, start=1):
+            template.id = i
+            template.save()
+
+    operations = [
+        migrations.AlterField(
+            model_name='emailtemplate',
+            name='slug',
+            field=models.SlugField(max_length=100, null=False, blank=False, unique=False, primary_key=False),
+        ),
+        migrations.AddField(
+            model_name='emailtemplate',
+            name='id',
+            field=models.AutoField(primary_key=True),
+        ),
+        migrations.RunPython(populate_template_ids),
+    ]
```

Comment:
tato migrace s daty bohužel nebude fungovat nemůžeš mít data bez primárního klíče, musí se to rozdělit do dvou migrací